client/src/client-only-routes/ShowProjectLinks.js

```diff
@@ -132,7 +135,12 @@ const ShowProjectLinks = props => {
               rel='noopener noreferrer'
               target='_blank'
             >
-              {cert.title}
+              <Trans
+                i18nKey={`certification.project.title.${cert.title}`}
+                defaults={cert.title}
+              >
+                placeholder
+              </Trans>
```

Comment:
For some reason bracket notation was not working for dynamic project title keys in `i18nKey`, so I went with the dot notation.

Reply:
It seems most of these `Trans` components can be changed to using the `t` function?

Reply:
A quick glance looks like only the footnote would need to use `Trans` I think. It works either way - the documentation recommends to use the `t` when you can.

Reply:
@ShaunSHamilton @moT01 for this case in particular I went with the `Trans` component because since the project title is dynamic I thought it would be safer to have `defaults` (in case the key is not found in `translations.json`) so that we can fall back to the original/untranslated project title. With the `t` function I guess I'd have to use ternary operator to make it work like this?

It's your call I can swap `Trans` component with the `t` function if you want except in the footnote as you said.

Reply:
How about ``t(`certification.project.title.${cert.title}`, cert.title)``, if the issue is defaults?

Reply:
Cool, I didn't know about this.